### ISSUE TITLE
THC notes and FU

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/clean#273]
Notes:
- No phone extensions
- Lots of time waiting on hold with MC, medicare, SSA (big one, federal number though) and healthy SF
- Most of their pop is on GA and/or SSI, so likely already enrolled in CF or not eligible.

To do:
- Schedule case worker info/training for ~April 1
- Liz will talk to case workers to find out if some resident groups may have higher eligibility rates
- Ask for intros to:
  - hospitality house
  - PHC + 'everyday connect'
  - SF Aids Foundation (needle exchange sites)
  - Mission Resource Center (Laura is ED - I met her in 2013)
  - Methadone clinics?
  - Shelter + Care > Liz will connect me with Salvador
  - New Roads = subsidized THC program for folks on probation
  - Compass Connecting Pt
  - Salvation Army - Sandra

**Comment by lippytak** [comment]
- send liz balance #s
